client/landing/stepper/declarative-flow/internals/steps-repository/intro/intro.tsx

```diff
@@ -4,25 +4,38 @@ import { useI18n } from '@wordpress/react-i18n';
 
 interface Props {
 	goNext: () => void;
-	flowName: string | null;
+	flowName: string;
 }
 
 const Intro: React.FC< Props > = ( { goNext, flowName } ) => {
 	const { __ } = useI18n();
 
-	const introTitle =
-		flowName === 'link-in-bio'
-			? createInterpolateElement( __( 'Let’s set up your<br />Link in Bio' ), { br: <br /> } )
-			: createInterpolateElement( __( 'Let’s set up your<br />Newsletter' ), { br: <br /> } );
+	const intro: any = {
+		newsletter: {
+			content: createInterpolateElement(
```

Comment:
really really minor but this will read better if its title instead of content

Reply:
You're right thanks. I also updated types 👍